src/content/resources/javascript/es6/arrow-functions.md

```diff
@@ -40,12 +41,31 @@ const sayHello = name => {
 };
 ```
 
-Another interesting feature that also reduces the syntax within arrow functions, is that if a function only returns an expression using one line of code, you can omit the `return` keyword and the curly braces. In arrow functions without braces, the `return` is implicit, meaning you don't need to include the `return` keyword. The following function returns "hello":
+Another interesting feature that also reduces the syntax within arrow functions, is that if the function is an expression, you can omit the `return` keyword and the curly braces. In arrow functions without braces, the `return` is implicit, meaning you don't need to include the `return` keyword. The following function returns "hello":
 
 ```js
 const sayHello = () => "hello";
 ```
 
+The same is also true of functions returning an expression using parameters, too.
+
+```js
+const sayHello = name => `hello ${name}`;
+```
+
+Or even...
+
+```js
+const helloObject = name => ({
+  isGreeting: true,
+  helloName: name
+});
+```
+
+This is utilising the parser precedence to wrap an object expression inside a grouped expression thus resulting in returning an object instead of expanding into a function body.
```

Comment:
Would drop the parser precedence stuff here. I only used it to explain what is actually happening. It's not that relevant and its filled with buzzwords

Reply:
Alright.